scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -171,7 +172,7 @@ final case class BigQueryTable(tableSpec: String) extends BigQueryIO[TableRow] {
     Reads.bqReadTable(sc)(beam.BigQueryIO.readTableRows(), table)
 
   override def write(data: SCollection[TableRow], params: WriteP): Tap[TableRow] = {
-    var transform = beam.BigQueryIO.writeTableRows().to(table)
+    var transform = beam.BigQueryIO.writeTableRows().to(table).withExtendedErrorInfo()
```

Comment:
```suggestion
    var transform = beam.BigQueryIO.writeTableRows().to(table)
```

I think `transform.withExtendedErrorInfo()` should be only set when `params.extendedErrorInfo` is `Enabled`?

And it is there in line 195.

Reply:
Ahh good catch! that wasn't supposed to be there 😄